src/util/Toaster.ts

```diff
@@ -0,0 +1,25 @@
+import * as vscode from 'vscode';
+import { DOWNLOAD_LOGS_CMD, DOWNLOAD_LOGS_TEXT } from '../common';
+
+/**
+ * Show toast messages to user.
+ */
+export class Toaster {
+  constructor() {}
+
+  showErrorMessage = async (message: string) => {
+    const response = await vscode.window.showErrorMessage(
+      message,
+      DOWNLOAD_LOGS_TEXT
+    );
+
+    // If user clicks "Download Logs" button
+    if (response === DOWNLOAD_LOGS_TEXT) {
+      await vscode.commands.executeCommand(DOWNLOAD_LOGS_CMD);
+    }
+  };
+
+  showInfoMessage = async (message: string) => {
```

Comment:
I'd probably just name these messages `error` and `info` respectively, similar to our `Logger` class. Up to you though.